src/olympia/addons/management/commands/process_addons.py

```diff
@@ -125,6 +127,21 @@ def get_recalc_needed_filters():
         'method': extract_colors_from_static_themes,
         'qs': [Q(type=amo.ADDON_STATICTHEME)]
     },
+    'delete_source_git_repositories': {
+        'method': delete_source_git_repositories,
+        'distinct': True,
+        'qs': [
+            Q(
+                # Retrieve a list of add-on IDs that have at least one version
+                # with a source git hash (which means source files have been
+                # git-extracted).
+                pk__in=Version.unfiltered.exclude(source_git_hash__exact='')
```

Comment:
@diox this feels weird but since the queryset for `process_addons` "starts from" the `Addon` model, I don't know how to write the query differently.

Reply:
It's a weird one but given what it's for I think that's fine, we shouldn't worry about spending too much time figuring out a better way